overlay.d/15fcos/usr/libexec/coreos-check-cgroups.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash
+# This script checks if the system is still using cgroupsv1
+# and prints a message to the serial console.
+
+# Change the output color to yellow
+warn=$(echo -e '\033[0;33m')
+# No color
+nc=$(echo -e '\033[0m')
+
+motd_path=/run/motd.d/30_cgroupsv1_warning.motd
+
+cat << EOF > "${motd_path}"
+${warn}
+############################################################################
+WARNING: This system is still using cgroupsv1.  It is strongly recommended
+to migrate this system and your workloads to use cgroupsv2.
+For instructions on how to remove kernel arguments for cgroupsv2, see:
+https://docs.fedoraproject.org/en-US/fedora-coreos/kernel-args/
+
+To disable this warning, use:
+sudo systemctl disable coreos-check-cgroups.service
```

Comment:
```suggestion
WARNING: This system is using cgroups v1. For increased reliability
it is strongly recommended to migrate this system and your workloads
to use cgroups v2. For instructions on how to adjust kernel arguments
to use cgroups v2, see:
https://docs.fedoraproject.org/en-US/fedora-coreos/kernel-args/

To disable this warning, use:
sudo systemctl disable coreos-check-cgroups.service
```

Optional wording enhancements. Feel free to use or ignore.